### PR TITLE
[TIMOB-23474] Android: Support titleid, messageid & okid in Ti.UI.AlertDialog (parity)

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-23474

### CODE REVIEW

File: android/modules/ui/src/java/ti/modules/titanium/ui/AlertDialogProxy.java [hunk 1]
```diff
@@ -23,8 +23,11 @@
 		TiC.PROPERTY_BUTTON_NAMES,
 		TiC.PROPERTY_CANCEL,
 		TiC.PROPERTY_MESSAGE,
+		TiC.PROPERTY_MESSAGEID,
 		TiC.PROPERTY_TITLE,
+		TiC.PROPERTY_TITLEID,
 		TiC.PROPERTY_OK,
+		TiC.PROPERTY_OKID,
```
Comment (hieupham007) [inline android/modules/ui/src/java/ti/modules/titanium/ui/AlertDialogProxy.java]: you may want to update the "since" for these properties

Comment (hansemannn) [inline android/modules/ui/src/java/ti/modules/titanium/ui/AlertDialogProxy.java]: Thought about that, but the properties themselves have been available before, just the getter/setter haven't. So how would we document that particular case?

**Comment by sgtcoolguy** [comment]
LGTM 👍

File: android/modules/ui/src/java/ti/modules/titanium/ui/AlertDialogProxy.java (hunk 1 above)
Comment (hieupham007) [inline android/modules/ui/src/java/ti/modules/titanium/ui/AlertDialogProxy.java]: probably easiest to assume they didn't work before, but works now.

**Comment by hansemannn** [comment]
@sgtcoolguy Feel free if you are good with it 😊

**Pull request merged**